.github/workflows/unit.yaml

```diff
@@ -15,4 +15,4 @@ jobs:
       - name: Run Test
         run: |
           make test
```

Comment:
Since you created `privileged-test` with a dependency of `tester`, why do we need this line?

Reply:
make test will run the regular - none privileged tests
make privileged-test will run the extra privileged tests. 
We just pushed line 18 into its own makefile target, so it will start all the needed dockers.